.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+Pull request for #PULL_REQUEST_ID
```

Comment:
I love the idea of having a PR template with a standard list of things to check off before each merge. I think this is a great jumping off point for such a checklist too. I think we should review this list as a group and see if we want to tweak it at all to best meet our goals as a project